.github/workflows/nightly_runner_master.yml

```diff
@@ -22,6 +22,8 @@ jobs:
               uses: actions/setup-node@v2
               with:
                   node-version: ${{ matrix.nodejs_version }}
+            - name: Checkout code
```

Comment:
This part already merged master. You can skip this file.

Reply:
can you merge master to your pr branch then?